packages/core/src/components/axes/axis.ts

```diff
@@ -462,6 +538,11 @@ export class Axis extends Component {
 					element: select(this),
 					datum,
 				});
+				if (!isTimeScaleType && axisOptions.scaleType && axisOptions.scaleType === "labels") {
+					self.services.events.dispatchEvent(Events.Tooltip.HIDE, {
```

Comment:
the tooltip hide event shouldn't need any detail or data being passed into it, is there a specific reason why this'd be needed?

Reply:
I can remove it. I saw that other tooltip hide events passed in data but didn't get used as well. I guess we can also clean it up.